jwst/resample/resample_step.py

```diff
@@ -195,128 +161,42 @@ def _load_custom_wcs(asdf_wcs_file, output_shape):
 
         return wcs
 
-    def get_drizpars(self, ref_filename, input_models):
+    def get_drizpars(self, input_models):
```

Comment:
Does this function use `input_models`?

Reply:
Hmmm, no it doesn't. Surprised style check didn't complain about that. I'll remove it. Good catch!

Reply:
Fixed in latest commit.